README.md

```diff
@@ -1,2 +1,62 @@
-# Portfolio-Mobile
-Personal portfolio in mobile format
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+My Personal Portfolio(Mobile)-Microverse
+
+> My Portfolio
+This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage.
+
+## Built With
+
+- HTML
+- CSS
+- FIGMA
+
+
+## Getting Started
+
+**Instructions on how to view and clone my project locally.**
+ To Access my project locally please follow the following instructions;
+1. Create a folder on your computer where you will clone the project
+2. Enter this url: https://github.com/dialite/Portfolio-Mobile/tree/Personal-portfolio in your web browser.
+3. Once opened, move your cursor to the top left of the project page to select 'code' a green code button will be visible on the righthand corner showing download    options.
+5. Select download Zip option from drop down menu to your desired folder on your computer.
+6. Once the download is complete you will be able to access my project locally.
+7. Also, you can navigate to your folder using a console.
+8. Then run git clone https://github.com/dialite/Portfolio-Mobile/tree/Personal-portfolio
+
+
+Here is a link directly to the repository page: https://github.com/dialite/Portfolio-Mobile/tree/Personal-portfolio
+
+### Basic Knowledge in HTML and CSS
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Raymond Olisa
+
```

Comment:
[OPTIONAL] Please remove the extra code from your README file. it is because of clean code.